app/views/appointments/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
+
+<h1>Appointments</h1>
+
+<table>
+  <thead>
+    <tr>
+      <th>Join</th>
+      <th>Leave</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @appointments.each do |appointment| %>
+      <tr>
+        <td><%= appointment.join %></td>
+        <td><%= appointment.leave %></td>
+        <td><%= link_to 'Show', appointment %></td>
+        <td><%= link_to 'Edit', edit_appointment_path(appointment) %></td>
+        <td><%= link_to 'Destroy', appointment, method: :delete, data: { confirm: 'Are you sure?' } %></td>
+      </tr>
```

Comment:
You are not providing a link to access this page. When I go directly on the browser, and I don't have an appointment, I can still see the links and that shouldn't happen.

Reply:
Done!